### PR TITLE
Injecting unordered lists of cookie projections per hour + total

### PR DESCRIPTION
**Pull request by dylanh4**
Use JS to inject unordered lists of projected cookies sold per location at each of the five Pat's Salmon Cookies locations as well as a final bullet at bottom of unordered list that provides a total of the cookies sold at that particular location.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1 +1,323 @@
 // Cookies needed per location projections
+
+var hours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm','6pm', '7pm', '8pm'];
```
Comment (nhuntwalker) [inline app.js:3]: You want the sales during the open hours. Note that the store isn't open from 8-9pm, so no need to include the 8pm hour.

File: sales.html [hunk 2]
```diff
@@ -1 +1,26 @@
...
+<!-- Internal Cookie Sales Tracking Application -->
+
+<html>
+  <head>
+    <title>Pat's Salmon Cookies: Inventory Projections</title>
+  </head>
+  <body>
+    <h1>Pat's Salmon Cookies: Inventory Projections</h1>
+    <h2>1st and Pike</h2>
+      <ul id="pikeID">
```
Comment (nhuntwalker) [inline sales.html:10]: These <ul> tags don't need to be indented under your <h2> tags.

File: sales.html [hunk 3]
```diff
@@ -1 +1,26 @@
...
+    <h2>Seattle Center</h2>
+      <ul id="seaCenterID">
+      </ul>
+    <h2>Capitol Hill</h2>
+      <ul id="capHillID">
+      </ul>
+    <h2>Alki</h2>
+      <ul id="alkiID">
+      </ul>
+  <script src="app.js"></script>
```
Comment (nhuntwalker) [inline sales.html:24]: This should be indented into your <body> tag.

File: app.js (hunk 1 above)
Comment (dylanh4) [inline app.js:3]: I replicated what was included in the assignment overview.

screenshot 2016-06-21 10 56 50

Should I not include 8pm given the text description or mirror what was displayed in the example (e.g. shows 8pm)?

**Pull request merged**